static/skin/viewer.js

```diff
@@ -262,10 +270,7 @@ function onClickEvent(e) {
   const target = matchingAncestorElement(e.target, iframeDocument, "a");
   if (target !== null && "href" in target) {
     if ( isExternalUrl(target.href) ) {
-      target.setAttribute("target", "_top");
-      if ( viewerSettings.linkBlockingEnabled ) {
-        return blockLink(target);
-      }
+      return blockLink(target);
```

Comment:
@veloman-yunkan Instead of loading a blocked page here, why not just do `window.open(url, target)` to open in a new window? What's the advantage of loading a blocked page instead., then opening the new window? (I I've understood correctly.) Also, consider using the `mousedown` event instead of `click`, as it happens before any click event listeners that might otherwise fire inside the iframe.

Reply:
> Instead of loading a blocked page here, why not just do window.open(url, target) to open in a new window? What's the advantage of loading a blocked page instead., then opening the new window? (I I've understood correctly.)

The external page is not opened in a new window. It is opened in the same browser window however in the top frame rather than in the viewer iframe.

> Also, consider using the mousedown event instead of click, as it happens before any click event listeners that might otherwise fire inside the iframe.

A link can also be activated via keyboard which I believe generates a `click` event but not a `mousedown` event.